src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -1116,13 +1089,28 @@ private void initConfigFromFlags(String[] args, PrintStream out, PrintStream err
       }
 
       // For backwards compatibility, allow both commonJsPathPrefix and jsModuleRoot.
-      List<String> moduleRoots = new ArrayList<>(flags.commonJsPathPrefix);
+      List<String> moduleRoots = new ArrayList<>();
       if (!flags.moduleRoot.isEmpty()) {
         moduleRoots.addAll(flags.moduleRoot);
       } else {
         moduleRoots.add(ES6ModuleLoader.DEFAULT_FILENAME_PREFIX);
       }
 
+      List<String> normalizedEntryPoints = new ArrayList<>();
+      for(String entryPoint : flags.entryPoints) {
+        if (entryPoint.startsWith("goog:")) {
+          normalizedEntryPoints.add(entryPoint.substring(5));
+        } else {
+          normalizedEntryPoints.add(ES6ModuleLoader.toModuleName(URI.create(entryPoint)));
```

Comment:
Nice. One question though - what's the error message the user will see when the expected entry point is missing? We should make sure it doesn't contain `$` all over the place, but rather reflects the original file name.

To make that simple, maybe we shouldn't be passing around `String`s, but some simple `ModuleIdentifier` that has both the original name (file or goog: symbol) and the mangled file name?